pos_loyalty/models/loyalty_reward.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2017 RGB Consulting S.L. (https://www.rgbconsulting.com)
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class LoyaltyReward(models.Model):
+    _name = 'loyalty.reward'
+
+    name = fields.Char(string='Reward Name', size=32, index=True,
+                       required=True)
+    type = fields.Selection(selection=[('gift', 'Gift'),
+                                       ('discount', 'Discount'),
+                                       ('resale', 'Resale')],
+                            string='Type', required=True,
+                            help='Type of the reward')
+    minimum_points = fields.Float(string='Minimum Points',
+                                  help='Minimum amount of points the customer'
+                                       ' must have to qualify for this reward')
+    point_cost = fields.Float(string='Point Cost',
+                              help='Cost of the reward per monetary unit '
+                                   'discounted')
+    discount = fields.Float(help='The discount percentage')
+    discount_max = fields.Float(string='Discount limit',
+                                help='Maximum discounted amount allowed for'
+                                     'this discount reward')
+    loyalty_program_id = fields.Many2one(comodel_name='loyalty.program',
+                                         string='Loyalty Program',
+                                         help='The Loyalty Program this reward'
+                                              ' belongs to')
+    gift_product_id = fields.Many2one(comodel_name='product.product',
+                                      string='Gift Product',
+                                      help='The product given as a reward')
+    discount_product_id = fields.Many2one(comodel_name='product.product',
+                                          string='Discount Product',
+                                          help='The product used to apply '
+                                               'discounts')
+    point_product_id = fields.Many2one(comodel_name='product.product',
```

Comment:
Shouln't these fields have a `pos_available = True` domain?

Reply:
You are right, only pos products should be available since it wont work otherwise.